### PR TITLE
RoleColor & GetSide accesibility

### PR DESCRIPTION
**Pull request by SebasCapo**
Added methods to get a RoleType's Color and more ways to access a Role/Team's Side

### CODE REVIEW

File: Exiled.API/Extensions/Role.cs [hunk 1]
```diff
@@ -5,13 +5,53 @@
 // </copyright>
 // -----------------------------------------------------------------------
 
+using Exiled.API.Enums;
```
Comment (galaxy119) [inline Exiled.API/Extensions/Role.cs:8]: Your using directives should be inside the namespace.

File: Exiled.API/Extensions/Role.cs [hunk 2]
```diff
@@ -5,13 +5,53 @@
...
+            role.GetTeam().GetSide();
+
+        /// <summary>
+        /// Get a <see cref="Team">team's</see> <see cref="Side"/>
+        /// </summary>
+        /// <param name="team"></param>
+        /// <returns></returns>
+        public static Side GetSide(this Team team) 
+            {
+            switch(team) {
```
Comment (galaxy119) [inline Exiled.API/Extensions/Role.cs:38]: The return messages are not indented properly.

Comment (SebasCapo) [reply Exiled.API/Extensions/Role.cs]: Fixed!

File: Exiled.API/Extensions/Role.cs (hunk 1 above)
Comment (SebasCapo) [reply Exiled.API/Extensions/Role.cs]: IHateThisButFixed!

**Comment by SebasCapo** [comment]
Should be better now!

**Pull request merged**